purchase_manual_delivery/models/purchase_order.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.tools import float_compare
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    pending_qty_to_receive = fields.Boolean(
+        compute="_compute_pending_qty_to_receive",
+    )
+
+    def _compute_pending_qty_to_receive(self):
+        for order in self:
+            order.pending_qty_to_receive = True
+            if all(
+                val is False
+                for val in order.mapped("order_line.pending_qty_to_receive")
+            ):
+                order.pending_qty_to_receive = False
+
+    @api.multi
+    def _create_picking(self):
+        # We do not want to create the picking when confirming the order
+        pass
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    existing_qty = fields.Float(
+        compute="_compute_existing_qty",
+        string="Existing Qty",
+        digits=dp.get_precision("Product Unit of Measure"),
+        help="Quantity already planned or shipped (stock movements "
+        "already created)",
+    )
+    pending_qty_to_receive = fields.Boolean(
```

Comment:
I would use:
```suggestion
    pending_to_receive = fields.Boolean(
```
I doesn't not make sense to me to use quantity on a boolean, could be confusing.

Reply:
Done :+1: